htdocs/comm/propal/list.php

```diff
@@ -1230,7 +1232,7 @@
 }
 if (!empty($arrayfields['s.nom']['checked'])) {
 	print '<td class="liste_titre" align="left">';
-	print '<input class="flat maxwidth100" type="text" name="search_societe" value="'.dol_escape_htmltag($search_societe).'">';
+	print '<input class="flat maxwidth100" type="text" name="search_societe" value="'.dol_escape_htmltag($search_societe).'"'.($socid > 0 ? " disabled" : "").'>';
```

Comment:
If field is disabled, the new search/submit of form will lost the filter (field will not be propagated).

Reply:
Yes exactly ,as it is done in facture/list.php.

When "filter" is send in QueryString with socid attribute, it is priorized with socid (the search_societe is displayed for the UX but not used anymore)